contracts/settling_game/L02_Resources.cairo

```diff
@@ -173,7 +177,7 @@ func claim_resources{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_che
 
     # SET MINT
     let treasury_mint_perc = wonder_tax
-    let user_mint_rel_perc = 100 - wonder_tax
+    let user_resources_value_rel_perc = 100 - wonder_tax
```

Comment:
@ponderingdemocritus I'm adding the following fix. Documenting this in case it breaks everything 🙃

```
    # Make sure we don't overflow
    let (less_than) = is_le(wonder_tax, 100)
    if less_than == TRUE:
        let (user_resources_value_relperc) = 100 - wonder_tax
    else:
        let (user_resources_value_relperc) = 0
```